app-office/heimer/heimer-1.10.0.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="A Qt Mind map application"
+DESCRIPTION="Desktop application for creating mind maps and other suitable diagrams.\
+It's written in Qt."
+COPYRIGHT="2018-2019 Jussi Lind"
+HOMEPAGE="https://github.com/juzzlin/Heimer"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/juzzlin/Heimer/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="d6850ab08825c2e5cb0278782a24a4fde8c16059bb6c2a23fbdd63f2952db0ff"
+SOURCE_FILENAME="Heimer-$portVersion.tar.gz"
+SOURCE_DIR="Heimer-$portVersion"
+ADDITIONAL_FILES="heimer.rdef.in"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+ SECONDARY_ARCHITECTURES="x86"
+fi
+
+PROVIDES="
+	heimer$secondaryArchSuffix = $portVersion
+	app:Heimer = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5PrintSupport$secondaryArchSuffix
+	lib:libQt5Svg$secondaryArchSuffix
+	lib:libQt5Xml$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5PrintSupport$secondaryArchSuffix
+	devel:libQt5Svg$secondaryArchSuffix
+	devel:libQt5Xml$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:qmake$secondaryArchSuffix >= 5
+	cmd:unzip
+	"
+
+BUILD()
+{
+	qmake
+	make $jobArgs
+}
+
+INSTALL()
+{
+	install -d "$appsDir"
```

Comment:
You can omit double quotes here and below.

Reply:
ok done